.github/CODEOWNERS

```diff
@@ -2,7 +2,7 @@
 # Each line is a file pattern followed by one or more owners.
 
 # These owners will be the default owners for everything in the repo.
-*       @ddietterich @RoriCremer @snf2ye @fboulnois @nmalfroy @pshapiro4broad @samanehsan @tlangs @tcjiang9
+*       @snf2ye @nmalfroy @pshapiro4broad @samanehsan @tlangs @tcjiang9 @okotsopoulos
```

Comment:
@ddietterich @RoriCremer @fboulnois please let me know if you'd like to be added back as codeowners.

Reply:
This is fine with me 👍